api/v1alpha/conduit_validator.go

```diff
@@ -1,18 +1,32 @@
 package v1alpha
 
 import (
+	"fmt"
+
 	"github.com/conduitio/conduit-operator/pkg/conduit"
 )
 
 var connectorValidators = []func(*ConduitConnector) error{
-	ValidatePlugin,
-	ValidatePluginType,
+	validateConnectorPlugin,
+	validateConnectorPluginType,
+}
+
+var processorValidators = []func(*ConduitProcessor) error{
+	validateProcessorPlugin,
 }
 
-func ValidatePlugin(c *ConduitConnector) error {
+func validateConnectorPlugin(c *ConduitConnector) error {
 	return conduit.ValidatePlugin(c.Plugin)
 }
 
-func ValidatePluginType(c *ConduitConnector) error {
+func validateConnectorPluginType(c *ConduitConnector) error {
 	return conduit.ValidatePluginType(c.Type)
 }
+
+func validateProcessorPlugin(p *ConduitProcessor) error {
+	if p.Plugin == "" {
+		return fmt.Errorf("plugin %q is invalid", p.Plugin)
```

Comment:
Nitpick: it would be nice if the error was more specific, e.g. "processor plugin empty".

Reply:
@hariso will do!

Reply:
Fixed here 666dd957084c3c724b9b975fd5dc1cc173265f97